crates/viewer/re_view_graph/src/ui/draw.rs

```diff
@@ -299,9 +299,10 @@ pub fn draw_graph(
 
                 let instance_path =
                     InstancePath::instance(entity_path.clone(), instance.instance_index);
-                ctx.select_hovered_on_click(
+                ctx.handle_select_hover_drag_interactions(
                     &response,
                     Item::DataResult(query.view_id, instance_path.clone()),
+                    false,
```

Comment:
Instead of adding `false` and `true` as argument, did you consider having two different functions, for increased readability? In particular, it is very hard for me reviewing this code to know what `false` means.

I.e. you could have kept the old `select_hovered_on_click` with the previous meaning, and added `handle_select_hover_drag_interactions` (without a `bool`) for draggable things. But maybe its better to force each users to consider wether or not an item is draggable in this context?

Reply:
Yes, I want people to think of draggability and make a call when using this function.

It's also a toggle that allows me to progressively implement draggability one UI area at a time (e.g. next PR will switch some items to `true` in the streams tree).

There is an ideal version of the future where this is `true` everywhere (and thus the arg can be removed), but I'm not sure this is realistic. There are cases where stuff should hover/select highlight, but not be draggable. For example, entity buttons should probably not be draggable—or at least not always (e.g. from the add/remove entity modal).

In generally, we should make something draggable only if it makes sense, e.g if there is an obvious drop destination.

I'll improve the docstring to expand on the above. Let me know if you have any further suggestion on this.